solana/ed25519/src/lib.rs

```diff
@@ -266,6 +266,13 @@ fn base64_display(bytes: &[u8], fmtr: &mut fmt::Formatter) -> fmt::Result {
     fmtr.write_str(unsafe { core::str::from_utf8_unchecked(&buf[..len]) })
 }
 
+/// Displays slice using base58 encoding.
+fn base58_display(bytes: &[u8], fmtr: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Perhaps `bytes: &[u8; 32]` and similarly in base64_display change to `bytes: &[u8; 64]`?

Reply:
sounds good.